src/motion/magic/utils.ts

```diff
@@ -36,7 +46,10 @@ export function snapshot(element: NativeElement): Snapshot {
         style: {
             backgroundColor,
             border,
-            borderRadius: borderRadius ? parseFloat(borderRadius) : 0,
+            borderTopLeftRadius: borderRadiusMatrix[0],
+            borderTopRightRadius: borderRadiusMatrix[1],
+            borderBottomLeftRadius: borderRadiusMatrix[2],
+            borderBottomRightRadius: borderRadiusMatrix[3],
```

Comment:
We could also combine this into a single array of border radii `[tl, tr, bl, br]` 🤔